public/resources/core/js/controllers/product-offering.controller.js

```diff
@@ -51,6 +51,34 @@
         vm.showFilters = showFilters;
         vm.getElementsLength = getElementsLength;
         vm.setFilters = setFilters;
+        vm.searchInput = "";
+
+        // Fill searchInput array
+        vm.fillSearchInput = fillSearchInput;
+        function fillSearchInput() {
```

Comment:
Are you using this method for something? is the same as getSearchInputContent but with a bug (using searchVM instead of vm)